ttnn/cpp/ttnn/operations/ccl/all_gather/device/multi_core/all_gather_op_multi_core.cpp

```diff
@@ -145,6 +145,8 @@ operation::ProgramWithCallbacks all_gather_multi_core_with_workers(const Tensor&
     std::unique_ptr<ccl::CclOpTensorConfig> output_tensor_config = ttnn::ccl::CclOpTensorConfig::build_all_gather_tensor_config(output_tensor);
 
     tt::tt_metal::Program program{};
+    // Issue #19078: CCLs need to be tagged as having multi-device dependencies, when running on Galaxy.
```

Comment:
this issue number doesn't exist. Also, don't we need to propagate this change across all CCL-ops?

Reply:
fyi @SeanNijjar

Reply:
Yes we will also need the same change on reduce scatter ideally.

Reply:
Comment updated.